app/Jobs/SendReportbackToPhoenix.php

```diff
@@ -33,14 +33,16 @@ public function handle()
     {
         $phoenix = new Phoenix;
 
+        $reportbackItem = $this->reportback->items()->orderBy('created_at', 'desc')->first();
```

Comment:
This is for sending a new reportback item back to Phoenix. Since you can only create on reportback at a time, every time a new reportback item is created, it will send back to Phoenix from Rogue. This code makes sure to take the most recently created (before it was taking the first reportback item created for that reportback and sending that information instead of the one that was just made).